consensus/src/dag/dag_store.rs

```diff
@@ -208,64 +74,42 @@ impl Dag {
                 .is_none(),
```

Comment:
existing code but, I think we should not replace here, otherwise even though we catch equivocation, by replacing, we might be accepting it?

Reply:
equivocation should not be possible with reliable broadcast, that's why I put it as assertion in the beginning, but sure, I can ensure it doesn't exist first